Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG base_image=ruby:2.7.2
```

Comment:
Current ruby version in main is now 2.7.5, so you might want to change this to:
`ARG base_image=ruby:2.7.5-slim-buster`